src/components/FormBadge/FormBadge.stories.tsx

```diff
@@ -1,43 +1,55 @@
-import type { Meta, StoryObj } from "storybook-framework-qwik";
-import { FormBadge, type FormBadgeProps } from "./FormBadge";
-import { CheckBox } from "../Checkbox/Checkbox";
-import { Input } from "../Input/Input";
 import { $ } from "@builder.io/qwik";
+import Checkbox from "../Atoms/Checkbox/Checkbox";
+import Input from "../Atoms/Input/Input";
+import { FormBadge, type FormBadgeProps } from "./FormBadge";
+import IconSuccess from "@material-design-icons/svg/round/check_circle_outline.svg?jsx";
+import { checkPattern } from "~/utils/fractions";
 
-const meta: Meta<FormBadgeProps> = {
+export default {
   component: FormBadge,
 };
 
-type Story = StoryObj<FormBadgeProps>;
+export function SelectToken(args: FormBadgeProps) {
+  return (
+    <FormBadge {...args}>
+      <Checkbox isChecked={true} />
+    </FormBadge>
+  );
+}
+
+export function ApprovalLimit(args: FormBadgeProps) {
+  const onInput = $((e: any) => {
+    const target = e.target;
+    const value = target.value;
+    console.log(value);
+    return value;
+  });
```

Comment:
Defined and never used, btw remove console.log

Reply:
Deleted